scripts/get

```diff
@@ -111,8 +111,10 @@ downloadFile() {
   HELM_DIST="helm-$TAG-$OS-$ARCH.tar.gz"
   DOWNLOAD_URL="https://kubernetes-helm.storage.googleapis.com/$HELM_DIST"
   CHECKSUM_URL="$DOWNLOAD_URL.sha256"
-  HELM_TMP_FILE="/tmp/$HELM_DIST"
-  HELM_SUM_FILE="/tmp/$HELM_DIST.sha256"
+  HELM_TMP_ROOT="/tmp/$(mktemp -d helm-installer-XXX)"
+  HELM_TMP_FILE="$HELM_TMP_ROOT/$HELM_DIST"
+  HELM_SUM_FILE="$HELM_TMP_ROOT/$HELM_DIST.sha256"
+  mkdir -p "$HELM_TMP_ROOT"
```

Comment:
this should be removed as per the above comment.

Reply:
Fixed.